File: calculator.py
sayi=(input("Bir sayı giriniz: "))
sayi=float(sayi)
sayi1=(input("İkinci Sayıyı Giriniz: "))
sayi1=float(sayi1)
islem=input("Yapmak İstediğiniz işlemi seçiniz(+,-,*,/,^):")
while islem not in ['+','-','*','/','^']:
    print("Geçersiz işlem Seçtniz.Lütfen Tekrara Deneyiniz")
    islem=input("Yapmak İstediğiniz işlemi seçiniz(+,-,*,/,^):")
if islem=='+':
    sonuc =sayi+sayi1
    print(f"{sayi}+{sayi1}={sonuc}")
elif islem=='-':
    sonuc=sayi-sayi1
    print(f"{sayi}-{sayi1}={sonuc}")
elif islem=='*':
    sonuc=sayi*sayi1
    print(f"{sayi}*{sayi1}={sonuc}")
elif islem=='/':
    if sayi1==0:
        print("Sonuç Tanımsızdır(Syntax Error)")
    else:
        sonuc=sayi/sayi1
        print(f"{sayi}/{sayi1}={sonuc}")
elif islem=='^':
    sonuc=sayi**sayi1
    print(f"{sayi}**{sayi1}={sonuc}")

File: calculator.java
import java.util.Scanner;
 public class calculator
 {
   public static void main(String[] args)
   {
     Scanner scan = new Scanner(System.in);
     System.out.println("İlk Sayıyı Giriniz:");
     double sayi=scan.nextDouble();
     System.out.println("İkinci Sayıyı Giriniz");
     double sayi1=scan.nextDouble();
     System.out.println("Yapmak İstediğiniz İşlemi Seçiniz(+,-,*,/):");
     String alinan_islem=scan.next();
     char islem=alinan_islem.charAt(0);
     double sonuc;
      if(islem=='+')
      {
        sonuc=sayi+sayi1;
        System.out.println("Sonuç:"+sonuc);
      }
      if (islem=='-')
      {
        sonuc=sayi-sayi1;
        System.out.println("Sonuç:"+sonuc);
      }
      if (islem=='*')
      {
        sonuc=sayi*sayi1;
        System.out.println("Sonuç:"+sonuc);
      }
      if (islem=='/')
      { 
        if(sayi1==0)
        {
            System.out.println("İşlem Tanımsızdır(Syntax Error:)");
        }
        else
        {
            sonuc=sayi/sayi1;
            System.out.println("Sonuç:"+sonuc);
        }
      }

     scan.close();

   }








 }